### PR TITLE
Add GMC icon

### LINKED ISSUE #11248: Request: GMC
### Brand Name

GMC

### Website

https://www.gmc.com/

### Popularity Metric

The similarweb rank is 12,861: https://www.similarweb.com/website/gmc.com/

### Official Resources for Icon and Color

SVG: https://www.gmc.com/content/dam/gmc/na/us/english/primary-nav-icons/GMC-153x28.svg
Color: `#E21836`

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by DevangGarg**
**Issue:** closes #11248.

**Popularity metric:**

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

File: slugs.md
Comment (adamrusted) [inline slugs.md:1]: SLUGS.md is an automatically generated file, you do not need to add anything here.

File: icons/gmc.svg
Comment (adamrusted) [inline icons/gmc.svg:1]: Though the viewbox is set to 24*24, it appears the path is still 'full size' and as such is outside the bounds of the viewbox. Please can you re-edit this to ensure the path is in the viewbox (without any transforms - the SVGO tool strips that out!)

File: _data/simple-icons.json [hunk 1]
```diff
@@ -6127,6 +6127,11 @@
             "hex": "EA4335",
             "source": "https://fonts.gstatic.com/s/i/productlogos/gmail_2020q4/v8/192px.svg"
         },
+        {
+            "title": "GMC",
+            "hex": "E21836",
+            "source": "https://www.gmc.com"
```
Comment (adamrusted) [inline _data/simple-icons.json:6133]: Notice on [this page](https://gm.com/copyright-trademark) suggests we cannot add this one.

**Review by adamrusted — CHANGES_REQUESTED** [review]
Thanks for your work on this @DevangGarg - there are a couple of points below, and I'd appreciate you completing the PR template in your post above to give us proper metrics for measuring popularity, etc.

**Comment by anhthang** [comment]
closes #11248

File: slugs.md
Comment (DevangGarg) [reply slugs.md:1]: I have removed it from pr.

File: icons/gmc.svg
Comment (DevangGarg) [reply icons/gmc.svg:1]: I have done the changes.

File: _data/simple-icons.json (hunk 1 above)
Comment (DevangGarg) [reply _data/simple-icons.json]: I have done the changes.

File: icons/gmc.svg
Comment (uncenter) [inline icons/gmc.svg:1]: This icon does not at all match the one from the URL listed as source.

File: _data/simple-icons.json [hunk 2]
```diff
@@ -6127,6 +6127,11 @@
             "hex": "EA4335",
             "source": "https://fonts.gstatic.com/s/i/productlogos/gmail_2020q4/v8/192px.svg"
         },
+        {
+            "title": "GMC",
+            "hex": "E21836",
+            "source": "https://www.gmc.com/content/dam/gmc/na/us/english/primary-nav-icons/GMC-153x28.svg"
```
Comment (uncenter) [inline _data/simple-icons.json:6133]: As per https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines we should avoid linking directly to the SVG files themselves and rather link to the page that has the SVG displayed on it.

**Review by uncenter — CHANGES_REQUESTED** [review]

**Comment by jrhaberland** [comment]
Depending on the answer from General Motors we may need to remove the Chevrolet, Cadillac, and GM Icon

**Comment by uncenter** [comment]
> Depending on the answer from General Motors we may need to remove the Chevrolet, Cadillac, and GM Icon

Correct. And #11247 would have to be closed.

**Comment by adamrusted** [comment]
Has someone asked for permission on this one yet?
Assuming not, can you please reach out to the brand @DevangGarg? You can find guidelines [here](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#requesting-permission).

**Comment by DevangGarg** [comment]
> Has someone asked for permission on this one yet? Assuming not, can you please reach out to the brand @DevangGarg? You can find guidelines [here](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#requesting-permission).

I think that General motors has said to remove the icon. Do I need to continue with this task?

**Comment by adamrusted** [comment]
Closing as no feedback from brand. If you do get official correspondence from them @DevangGarg - feel free to post it here and we can re-open the PR.

**Pull request closed without merging**